src/app/components/AddressBox/__tests__/index.test.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import copy from 'copy-to-clipboard'
+import { screen } from '@testing-library/dom'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import { AddressBox } from '../index'
+
+jest.mock('copy-to-clipboard')
+
+const testAddress = 'oasis1qqurxkgavtcjjytneumeclx59ds3avjaqg7ftqph'
+
+const renderComponent = () => render(<AddressBox address={testAddress} />)
+
+describe('<AddressBox />', () => {
+  it('should render address properly', () => {
+    const { container } = renderComponent()
+    expect(container).toMatchSnapshot()
+  })
+
+  it('should be able to copy address to clipboard', async () => {
+    renderComponent()
+    await userEvent.click(screen.getByTestId('copy-address'))
+    expect(copy).toHaveBeenCalledWith(testAddress)
+  })
+
+  it('should be able to show a notification', async () => {
+    renderComponent()
+    jest.mocked(copy).mockReturnValue(true) // Copy must return true so that the notification is actually displayed
+    await userEvent.click(screen.getByTestId('copy-address'))
+    expect(await screen.getByTestId('copy-notification')).toBeInTheDocument()
```

Comment:
To follow https://testing-library.com/docs/queries/about/#priority, and because it's quite easy to find the notification without testId

```suggestion
    expect(await screen.getByText('account.addressCopied')).toBeInTheDocument()
```

Reply:
OK, fixed.